lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss

```diff
@@ -26,22 +26,23 @@
       border-top: 2px solid $light-gray;
     }
 
-    > .wrapper {
-      display: none;
+    .social-icon {
+      padding: $base-spacing / 2;
 
       @include media($tablet) {
-        display: block;
+        padding: $base-spacing;
       }
     }
+  }
 
-    .social-icon {
-      padding: $base-spacing / 2;
+  .card__copy {
+    .-share {
```

Comment:
We need modifiers to be directly attached to a class to avoid naming conflicts, so I'd change this to `&.-share` and then maybe style any `h3` or `p` children.